charts/rancher-gke-operator/1.0.200/questions.yaml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.5.8-alpha1
+rancher_min_version: 2.5.8-patch1
```

Comment:
gke-operator 1.0.200 is the version that will be released in rancher v2.5.10 in future